src/services/swap_service.ts

```diff
@@ -479,10 +492,11 @@ export class SwapService {
                 assetSwapperOpts,
             );
         } else if (buyAmount !== undefined) {
+            const buyAmountScaled = buyAmount.dividedToIntegerBy(1 - affiliateFee.buyTokenPercentFee);
```

Comment:
hmm I was thinking 
```suggestion
            const buyAmountScaled = buyAmount.plus(buyAmount.times(affiliateFee.buyTokenPercentFee))
```

So a 100 buy 50% fee would
```
100 + 100 * 0.5 = 150 
```

versus
```
100 / (1 - 0.5) = 200 
```

unless I'm reading this wrong?

Reply:
ok yeah